tel2puml/pv_to_puml.py

```diff
@@ -255,26 +255,90 @@ def pv_events_from_folder_to_puml_file(
 
 def pv_streams_to_puml_files(
     pv_streams: Generator[
```

Comment:
All of the Generator types can be an Iterable here as this function is as general as that